src/utils/versioning.js

```diff
@@ -1,22 +1,16 @@
 const { isProduction } = require('./utils.node');
 /** version number for which documentation is available (except for the latest version) */
 const ALL_SUPPORTED_JS_API_VERSIONS = [
-  'v0.32',
```

Comment:
@olegbespalov, I am not 100% sure but I think this will fail in `production`. 
Please, do not remove old version from the list as we'll handle this on a future PR.

Reply:
Sure, reverted https://github.com/grafana/k6-docs/pull/1139/files#diff-5382900d75c5728bb60cb66cbc1678862ed6465fb96893135f6a9a6d16ed31ca